### PR TITLE
Reorg docs and fix css for sphix-gallery examples

### PR DESCRIPTION
**Pull request by narendasan**
# Description

Reorgs some files and fixes the CSS for the gallery

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Review by gs-olive — COMMENTED** [review]
The changes improve the organization and look of the Tutorials greatly. I wonder if the thumbnails could be adapted to include some sort of download/other data instead of linking to the same hyperlinked pages as above. See the bottom of this [scikit-learn examples page](https://scikit-learn.org/stable/auto_examples/index.html), for instance.

File: examples/dynamo/README.rst [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-This document contains examples of usage of the `torch_tensorrt.dynamo.compile` API which integrates with `torch.compile` functionality
+Torch-TensorRT provides a backend for the new ``torch.compile`` API released in PyTorch 2.0. In the following examples we describe
+a number of ways you can leverage this backend to accelerate inference.
 
-Overview of Available Scripts
------------------------------------------------
-- `dynamo_compile_resnet_example.py <./dynamo_compile_resnet_example.html>`_: Example showcasing compilation of ResNet model
-- `dynamo_compile_transformers_example.py <./dynamo_compile_transformers_example.html>`_: Example showcasing compilation of transformer-based model
-- `dynamo_compile_advanced_usage.py <./dynamo_compile_advanced_usage.html>`_: Advanced usage including making a custom backend to use directly with the `torch.compile` API
+* :ref:`dynamo_compile_resnet`: Compiling a ResNet model using the Dyanmo Compile Frontend for ``torch_tensorrt.compile``
```
[the remark this answers is not in the document]
Comment (narendasan) [reply examples/dynamo/README.rst:9]: The example uses torch_tensorrt.compile right?

File: examples/dynamo/README.rst [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+* :ref:`torch_compile_transformer`: Compiling a Transformer model using ``torch.compile``
```
[the remark this answers is not in the document]
Comment (narendasan) [reply examples/dynamo/README.rst:10]: Id like for the Transformer script to use the torch.compile interface

File: examples/dynamo/README.rst [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-This document contains examples of usage of the `torch_tensorrt.dynamo.compile` API which integrates with `torch.compile` functionality
+Torch-TensorRT provides a backend for the new ``torch.compile`` API released in PyTorch 2.0. In the following examples we describe
+a number of ways you can leverage this backend to accelerate inference.
 
-Overview of Available Scripts
------------------------------------------------
-- `dynamo_compile_resnet_example.py <./dynamo_compile_resnet_example.html>`_: Example showcasing compilation of ResNet model
-- `dynamo_compile_transformers_example.py <./dynamo_compile_transformers_example.html>`_: Example showcasing compilation of transformer-based model
-- `dynamo_compile_advanced_usage.py <./dynamo_compile_advanced_usage.html>`_: Advanced usage including making a custom backend to use directly with the `torch.compile` API
+* :ref:`dynamo_compile_resnet`: Compiling a ResNet model using the Dynamo Compile Frontend for ``torch_tensorrt.compile``
```
Comment (gs-olive) [inline examples/dynamo/README.rst:9]: Needs rename, `Dynamo Compile` now means something else in the `torch_tensorrt` context

**Pull request merged**